rio/src/RIO/Prelude/Partial.hs

```diff
@@ -0,0 +1,8 @@
+module RIO.Prelude.Partial
```

Comment:
Sorry I missed this last time around, but we don't need this layer of indirection. The RIO.Prelude modules are intended for this exported by RIO itself